PlayCover/ViewModel/StoreVM.swift

```diff
@@ -20,23 +19,43 @@ class StoreVM: ObservableObject, @unchecked Sendable {
         resolveSources()
     }
 
+    @Published var enabledList: [String] = (UserDefaults.standard.stringArray(forKey: "enableSourceList") ??
+                                             StoreVM.shared.sourcesList.map { $0.source })
+
     @Published var sourcesList: [SourceData] {
         didSet {
             encode()
         }
     }
     @Published var sourcesData: [SourceJSON] = [] {
         didSet {
-            sourcesApps.removeAll()
-            for source in sourcesData {
-                appendSourceData(source)
-            }
+            updateSourcesApps()
         }
     }
     @Published var sourcesApps: [SourceAppsData] = []
 
     private var resolveTask: Task<Void, Never>?
 
+    //
+    func updateSourcesApps() {
+        sourcesApps.removeAll()
+        let enabledSources: [SourceJSON] = sourcesData.filter { enabledList.contains($0.sourceURL) }
+          for source in enabledSources {
+                appendSourceData(source)
+            }
+    }
+
+    //
+    func enableSourceToggle(source: SourceData, value: Bool) {
+        if enabledList.contains(source.source) && !value {
+            enabledList.removeFirstObject(object: source.source)
```

Comment:
Something like `enabledList.filter { $0 != source.source }` but might have to compare a specific value within SourceData.